java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobFinishedEvent.java

```diff
@@ -21,6 +21,9 @@
 
 import java.util.Objects;
 
+/**
+ * A event class for when an ingest job was finished. Used in the ingest job status store.
```

Comment:
We can omit "class" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html